tests/test_asset_scanner.py

```diff
@@ -0,0 +1,62 @@
+import ast
+import os
+import sys
+from io import StringIO
+
+from asset_scanner.scripts import asset_scanner
+
+extract_assets_conf = os.path.join('conf', 'extract-assets.yaml')
+extract_items_conf = os.path.join('conf', 'extract-items.yaml')
+
+
+class Capturing(list):
```

Comment:
@Mahir-Sparkess could you not just use pytest's `capsys`

Reply:
Thanks, that's very useful to know